spec/helpers/session.rb

```diff
@@ -0,0 +1,33 @@
+module SessionHelpers
+
+  def sign_up (name: 'Edward Kerry',
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)